cmd/version.go

```diff
@@ -1 +1,68 @@
 package cmd
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+)
+
+var Version = "0.0.0"
```

Comment:
Minor: Is there a reason this is exported?

Reply:
Probably not while it is only used in verify.go although it is the source of truth for the version so like it being available.